apps/website/src/components/chart/components/chart-component.tsx

```diff
@@ -0,0 +1,231 @@
+import { useMemo } from 'react'
+
+import { animated } from '@react-spring/web'
+import { Axis } from '@visx/axis'
+import { GridColumns } from '@visx/grid'
+import { Group } from '@visx/group'
+import { scaleLinear, scaleTime } from '@visx/scale'
+import { extent, max } from 'd3-array'
+
+import { getClosedIssues, getTotalIssues } from '../helpers/get-data'
+import { useAnimations, useChartTooltip } from '../hooks/'
+import { format } from '../utils/format-time'
+import { Areas } from './areas'
+import { ChartTooltip } from './chart-tooltip'
+import { Lines } from './lines'
+import { Markers } from './markers'
+
+import type { ChartProps } from '../chart'
+
+// defining colors
+const colors = {
+  total: '#E95460',
+  closed: '#23ADA0',
+  background: '#F0F2F5',
+  marker: '#09101C',
+  totalGradient: 'rgba(233, 84, 96, 0.3)',
+  closedGradient: 'rgba(35, 173, 160, 0.3)',
+  white: '#FFF',
+}
+
```

Comment:
I recommend doing this
```
const colors = {
  total: '#E95460',
  closed: '#23ADA0',
  background: '#F0F2F5',
  marker: '#09101C',
  totalGradient: 'rgba(233, 84, 96, 0.3)',
  closedGradient: 'rgba(35, 173, 160, 0.3)',
  white: '#FFF',
} as const

export type Colors = typeof colors
```

and use `Colors` type to type the props of the components.

Reply:
Actually, you don't even need to pass it as via props because these data are static. Just import the colors object.